server/src/main/java/org/elasticsearch/index/mapper/BooleanFieldMapper.java

```diff
@@ -142,7 +142,12 @@ private FieldValues<Boolean> scriptValues() {
         }
     }
 
-    public static final TypeParser PARSER = new TypeParser((n, c) -> new Builder(n, c.scriptCompiler(), c.indexVersionCreated()));
+    private static final Version minimumCompatibilityVersion = Version.fromString("5.0.0");
```

Comment:
nit: I think static final vars should be in all-caps? ie `MINIMUM_COMPATIBILITY_VERSION`